src/features/user/MFV2/MyContributions/ProjectHeader.tsx

```diff
@@ -1,5 +1,5 @@
 import styles from './MyContributions.module.scss';
-import ProjectTypeIcon from '../../../projectsV2/components/microComponents/ProjectTypeIcon';
+import ProjectTypeIcon from '../../../projectsV2/ProjectSnippet/microComponents/ProjectTypeIcon';
 import { useTranslations } from 'next-intl';
```

Comment:
If ProjectTypeIcon is needed in multiple places, relocate it appropriately instead of leaving it within ProjectSnippet

Reply:
moved it to common folder